stable_baselines/deepq/build_graph.py

```diff
@@ -136,47 +134,40 @@ def build_act(make_obs_ph, q_func, num_actions, scope="deepq", reuse=None):
                 should be passed to outer variable scope
         and returns a tensor of shape (batch_size, num_actions) with values of every action.
     :param num_actions: (int) number of actions.
-    :param scope: (str or VariableScope) optional scope for variable_scope.
-    :param reuse: (bool) whether or not the variables should be reused. To be able to reuse the scope must be given.
+    :param observations_ph: (TensorFlow Tensor) the observation placeholder
+    :param stochastic_ph: (TensorFlow Tensor) the stochastic placeholder
```

Comment:
?

Reply:
it changes nothing to the behaviour, it is only so the input placeholder is within the scope "input" so the graph is cleaner